### ISSUE TITLE
Support `crashed` session events in Ruby SDK

### ISSUE THREAD

**Issue by Fwang36** [getsentry/sentry-ruby#2426]
Currently, we are not sending the crashed session event type in the Ruby SDK.  This leads to users of the Ruby SDK and the Release Health feature to always have 100% Crash Free Rate, even if the errors themselves are marked as unhandled.

We should support sending crashed session events for unhandled errors so Release Health works properly

**Comment by sandstrom** [comment]
Don't know if this is the right place, but would be great if the Ruby SDK supported the release health stuff.

https://docs.sentry.io/product/releases/setup/

According to this doc page it's not yet supported in the Ruby SDK.

Would also be good if we could mark errors as unhandled, which [seems to be supported](https://github.com/getsentry/sentry-ruby/pull/2280) by it's not clearly documented.

Basically, we are catching exceptions, rendering our own 5xx error page and would like to mark them as 'unhandled' in our call to `capture_exception`, e.g. something like `Sentry.capture_exception(…, unhandled: true)`
